_data/sites.json

```
@@ -10373,6 +10373,15 @@
         ]
     },
 
+    {
+        "name": "YoYo Games",
+        "url": "https://help.yoyogames.com/hc/en-us/articles/360025895752-How-can-I-delete-my-data-",
+        "difficulty": "hard",
```

```suggestion
        "difficulty": "hard",
        "email": "datacontroller@yoyogames.com",
```

From the link provided, it seems like the best way to go is to send an email